src/DOM/HTML/URL.purs

```diff
@@ -0,0 +1,14 @@
+module DOM.HTML.URL
+  ( createObjectURL
+  , revokeObjectURL
+  ) where
+
+import Control.Monad.Eff (Eff)
+import DOM (DOM)
+import DOM.File.Types (File)
+import DOM.HTML.Types (URL)
+import Data.Unit (Unit)
+
+foreign import createObjectURL :: forall eff. URL -> File -> Eff (dom :: DOM | eff) String
```

Comment:
Make URL the last argument, perhaps, to make it easier to use like this:

```
fileObjectUrl <- createObjectURL someFile =<< url =<< window
```

Discussed here also: https://github.com/purescript-contrib/purescript-dom/pull/75